### PR TITLE
Add assigned repositories list [SCI-4507]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-4507](https://biosistemika.atlassian.net/browse/SCI-4507)

### What was done
Add assigned repositories list

### CODE REVIEW

File: app/models/my_module.rb [hunk 1]
```diff
@@ -205,6 +205,15 @@ def repository_rows_count(repository)
                              .count
   end
 
+  def assigned_repositories_list
```
Comment (okriuchykhin) [inline app/models/my_module.rb:208]: Does it return collection of Repository models?

Comment (aignatov-bio) [reply app/models/my_module.rb]: Updated

File: app/models/my_module.rb [hunk 2]
```diff
@@ -205,6 +205,13 @@ def repository_rows_count(repository)
                              .count
   end
 
+  def assigned_repositories
```
Comment (okriuchykhin) [inline app/models/my_module.rb:208]: it will do 2 separate SQL queries

File: app/views/my_modules/protocols.html.erb [hunk 3]
```diff
@@ -39,7 +39,7 @@
     </a>
     </div>
     <div class="collapse" id="assigned-items-container">
-      ...
+      <%= render partial: "my_modules/inventories/inventories_list" %>
```
Comment (mlorb) [inline app/views/my_modules/protocols.html.erb:42]: In code we mostly use `repositories`. Can you rename partial.

Comment (aignatov-bio) [reply app/views/my_modules/protocols.html.erb]: updated

File: app/models/my_module.rb (hunk 2 above)
Comment (aignatov-bio) [reply app/models/my_module.rb:208]: updated

**Pull request merged**